crates/libs/targets/Cargo.toml

```diff
@@ -24,8 +24,8 @@ windows_i686_gnu = { path = "../../targets/i686_gnu", version = "0.48.0" }
 [target.'cfg(all(target_arch = "x86_64", target_env = "gnu", not(target_abi = "llvm"), not(windows_raw_dylib)))'.dependencies]
```

Comment:
Left `not(target_abi = "llvm")` because it doesn't break the build and might be necessary one day.